tests/line.test.js

```diff
@@ -0,0 +1,13 @@
+const expect = require('chai').expect;
+const Line = require('../lib/line.js');
+const Cell = require('../lib/cell.js');
+
+it('identifies a win', () => {
+  let line = new Line(new Cell(1), new Cell(2), new Cell(3));
+
+  line.cells.forEach((cell) => {
```

Comment:
I would format that the same way that you did in previous classes
line.cells.forEach(cell => cell.markWithSymbol('X'));